app/views/layouts/application.html.erb

```diff
@@ -2,15 +2,36 @@
 <html>
 <head>
   <title>Bars</title>
+  <title><%= content_for?(:title) ? yield(:title) : "foundation-rails" %></title>
+
   <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track" => true %>
-  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
+  <%= javascript_include_tag "vendor/modernizr" %>
   <%= csrf_meta_tags %>
 </head>
 <body>
-<p class="notice"><%= notice %></p>
-<p class="alert"><%= alert %></p>
+  <%= button_to "Sign in", new_user_session_path %>
+  <%= button_to "Sign out", destroy_user_session_path, method: :delete %>
+  <%= button_to "Edit profile", edit_user_registration_path, method: :get %>
+  <header>
+    <!-- <p class="notice"><%= notice %></p>
```

Comment:
Remove commented out code! Esp. in ERB, commented out code tends to break things.